.github/workflows/build_and_test.yml

```diff
@@ -217,6 +225,7 @@ jobs:
           source ./scripts/init_env.sh && ./scripts/${{ matrix.demo_script }}
 
       - name: Upload logs
+        continue-on-error: true
```

Comment:
The log-uploads have been extremely flaky recently, but this shouldn't trigger a fail in the CI.